### DISCUSSION TITLE
Playback too slow on a M5Stack Core 2 (AWS version)

### DISCUSSION THREAD

**Discussion by epollari** [schreibfaul1/ESP32-audioI2S#356] [General]
Audio playback on my M5Stack Core 2 (orange AWS version) is noticeably slow. I used the M5Core2.ino example for testing. The CPU frequency in the Arduino IDE was already maxed out at 240MHz, but I tried setting it to 160Mhz. Unsurprisingly, playback speed is slower still.

Any ideas or fixes?

**Comment by daywink** [comment]
Hello, I have the same issue with the original white Core2 module. The issue seems to be the «M5Core2» library is making problems, as with it removed I have no issues.
